src/browser/cacheDetail.js

```diff
@@ -15,6 +15,25 @@ function CacheDetailController($scope, SimDriver) {
     ctrl.activeCache = 'L1';
     ctrl.maq = [];
     ctrl.index = 0;
+    //ctrl.modals = {};
+    ctrl.modals = {
+        'L1' : {
+            'modal-01' : {
+                'tag' : 10110,
+                'block' : [...Array(32).keys()].map(x => x+2000),
```

Comment:
what's this

Reply:
just dummy data to show what an individual cell in the cache details page would contain.  It has a modal element per cell which contains all the information associated with the given block stored in that cell

Reply:
Will be replaced by the information given from the simulation state